chia/rpc/wallet_rpc_api.py

```diff
@@ -1233,15 +1234,39 @@ async def verify_signature(self, request) -> EndpointResult:
         :param request:
         :return:
         """
+        input_message: str = request["message"]
+        signing_mode_str: Optional[str] = request.get("signing_mode")
+        # Default to BLS_MESSAGE_AUGMENTATION_HEX_INPUT as this RPC was originally designed to verify
+        # signatures made by `chia keys sign`, which uses BLS_MESSAGE_AUGMENTATION_HEX_INPUT
+        signing_mode = (
+            SigningMode(signing_mode_str)
```

Comment:
I believe this will throw/raise some enum exception here if `signing_mode_str` is not one of the enums values - is that what you want?

Reply:
It does provide at least a legible error message.

```python-console
Python 3.10.8 (main, Oct 15 2022, 17:48:51) [GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import enum
>>> class E(enum.Enum):
...     ab = "ab"
...     cd = "cd"
... 
>>> E("ab")
<E.ab: 'ab'>
>>> E("cd")
<E.cd: 'cd'>
>>> E("ef")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/altendky/.pyenv/versions/3.10.8/lib/python3.10/enum.py", line 385, in __call__
    return cls.__new__(cls, value)
  File "/home/altendky/.pyenv/versions/3.10.8/lib/python3.10/enum.py", line 710, in __new__
    raise ve_exc
ValueError: 'ef' is not a valid E
```

Reply:
Yes, this was deliberate. I tried the same experiment as @altendky and felt it was a reasonable error message. But perhaps it's asking a bit much to rely on this behavior. I can wrap the call in a try/except and provide our own error message.